esx_service/vmdk_ops.py

```diff
@@ -584,6 +584,10 @@ def executeRequest(vm_uuid, vm_name, config_path, cmd, full_vol_name, opts):
                    "Default datastore: %s" \
                    % (datastore, ", ".join(known_datastores()), vm_datastore))
 
+    error_info, tenant_uuid, tenant_name = auth.authorize(vm_uuid, datastore, cmd, opts)
+    if error_info:
+        return err(error_info)
+
```

Comment:
@govint - we do not have pylint in the build and no Py3 in CI test bed, so the py3 compatibility can rot. I will add pylint and @kerneltime - can we add py3 (latest ESX GA) to the CI ?